internal/constants/status.go

```diff
@@ -0,0 +1,17 @@
+package constants
+
+import "github.com/qdm12/ddns-updater/internal/models"
+
+const (
+	FAIL     models.Status = "failure"
+	SUCCESS  models.Status = "success"
+	UPTODATE models.Status = "up to date"
+	UPDATING models.Status = "updating"
+)
+
+const (
+	HTML_FAIL     string = `<font color="red"><b>Failure</b></font>`
```

Comment:
Yeah, agreed. I kept it there basically to remind me to replace it with a type alias for HTML strings, as I have a bunch of HTML strings and it would be nice not to mix them around when coding with normal strings. But good catch, I'm a horrible person not commenting my code as well with TODOs 😄